test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/HtmlDocumentTest/AcceptsEndTagWithNoMatchingStartTag.stree.txt

```diff
@@ -1,13 +1,14 @@
-Markup block - Gen<None> - 14 - (0:0,0)
-    Markup span - Gen<Markup> - [Foo ] - SpanEditHandler;Accepts:Any - (0:0,0) - Tokens:2
-        SyntaxKind.Text;[Foo];
-        SyntaxKind.Whitespace;[ ];
-    Tag block - Gen<None> - 6 - (4:0,4)
-        Markup span - Gen<Markup> - [</div>] - SpanEditHandler;Accepts:Any - (4:0,4) - Tokens:4
-            SyntaxKind.OpenAngle;[<];
-            SyntaxKind.ForwardSlash;[/];
-            SyntaxKind.Text;[div];
-            SyntaxKind.CloseAngle;[>];
-    Markup span - Gen<Markup> - [ Bar] - SpanEditHandler;Accepts:Any - (10:0,10) - Tokens:2
-        SyntaxKind.Whitespace;[ ];
-        SyntaxKind.Text;[Bar];
+SyntaxKind.HtmlDocument - [0..14) - FullWidth: 14 - Slots: 1 - [Foo </div> Bar]
+    SyntaxKind.HtmlMarkupBlock - [0..14) - FullWidth: 14 - Slots: 1
+        SyntaxKind.HtmlTextLiteral - [0..4) - FullWidth: 4 - Slots: 1 - Gen<Markup> - SpanEditHandler;Accepts:Any
```

Comment:
I've always felt like these things are kinda gibberish to me `Gen<Markup> - SpanEditHandler;Accepts:Any` - maybe I just never learned how to read it.

Reply:
`Gen<Markup>` is just the chunk generator which will go away. `SpanEditHandler;Accepts:Any` is just the `EditHandler` type and the `AcceptedCharacters`